Extension/package.json

```diff
@@ -2209,13 +2224,23 @@
           "command": "workbench.action.showAllSymbols",
           "group": "other1_navigation@4"
         },
+        {
+          "when": "editorLangId == cuda-cpp",
```

Comment:
interesting that we add an object for each languageId.  I think we can do `||` in these `when` blocks, but that can be something we look into for a subsequent PR.

Reply:
Can we just try that now and see if it works? I recall I tried a long time ago and it didn't work but may be it works now?

Reply:
Could we defer that to a future PR, just in case?

Reply:
Isn't it easy to check to see if it works? I tried and it works. The keyboard command case needs to be formatted like `editorLangId == 'cpp' && editorTextFocus || editorLangId == 'c' && editorTextFocus` without `()` grouping.

Reply:
Oh, did you not change the keyboard switch header case? Is that supposed to work with CUDA? I see an issue at https://github.com/microsoft/vscode-cpptools/issues/4509

Reply:
Seems like the keybinding should be updated since the context menus exist.